doc/manuals/orion-api.md

```diff
@@ -2816,15 +2816,25 @@ _**Request headers**_
 _**Response code**_
 
 * Successful operation uses 200 OK
-* Errors use a non-2xx and (optionally) an error payload. See subsection on [Error Responses](#error-responses) for more details.
+* Errors use a non-2xx code and error payload (see next subsection):
```

Comment:
Should the link to the error responses section be included? (applies for each API operation)

Reply:
The subsection is

```
_**Response payload**_
```

That as far as a I understand, is not link-able.